crates/gosub_cairo/src/scene.rs

```diff
@@ -0,0 +1,278 @@
+use crate::elements::border::GsBorderRadius;
+use crate::elements::rect::GsRect;
+use crate::elements::text::GsText;
+use crate::elements::transform::GsTransform;
+use crate::CairoBackend;
+use gosub_shared::render_backend::{
+    Point, Radius, RenderBackend, RenderRect, RenderText, Scene as TScene, Transform as TTransform, FP,
+};
+use std::fmt::{Debug, Formatter};
+
+/// A scene command that can be executed onto a cairo context.
+#[derive(Clone)]
+pub enum SceneCommand {
+    // Draw a rectangle, including rounded corners and border
+    Rectangle(Box<RenderRect<CairoBackend>>),
+    // Draw a text
+    Text(Box<RenderText<CairoBackend>>),
```

Comment:
Is it really worth it to put those inside a Box? I mean, we will mainly use these, so it should be fine if the size of `SceneCommand` is as big as the `RenderRect` or `RenderText` itself are

Reply:
yes.. otherwise the enum differences are too big (800 bytes, vs 400 bytes, vs a lot less)... cargo complains about this

Reply:
Yes, but I assume 95% of the scene commands are Rectangle and Text Nodes, if we have another one, so SimpleText or Group, we are wasting a bit of memory, but it probably is worth it